status-im/status-react
# Fix reaction images and implement Selectors > Reactions component

Fixes https://github.com/status-im/status-mobile/issues/16045

### Summary

This PR fixes reaction images and implements the component `Selectors > Reactions` that, for some reason, wasn't implemented as a separate quo2 component as per [Figma](https://www.figma.com/file/qLLuMLfpGxK9OfpIavwsmK/Iconset?type=design&node-id=942-218&t=cqTr12Q3zVHaLoap-0).

#### Component

<img src="https://github.com/status-im/status-mobile/assets/46027/921bddf5-62a5-41b8-8f29-b11c652109f2" height="400" />

<img src="https://github.com/status-im/status-mobile/assets/46027/109be87b-de88-4a2a-a51d-39f7d4bf415e" height="250" />


#### Drawer

<table>
  <thead>
    <tr>
      <th>🐛🐜</th>
      <th>Fixed</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><img src="https://github.com/status-im/status-mobile/assets/46027/d8a04fe5-66ab-4651-bae2-9c332e7e9914" height="600" /></td>
      <td><img src="https://github.com/status-im/status-mobile/assets/46027/43181875-c9a5-479c-b8e9-fdbef3c79f7a" height="600" /></td>
    </tr>
  </tbody>
</table>

### Review notes

Reaction images in the Design System are not icons, so that's why you are seeing a bunch of icons removed from `icons2`. The directory `resources/images/reactions` already existed, and so I used images from that directory instead.

#### Platforms

- Android
- iOS

### Steps to test

- Open any chat and try to react. Try also to open the message drawer to see the fixed reactions.
- You can also check out the previews in `Selectors > reactions`.

status: ready
